src/client/index.ts

```diff
@@ -133,6 +133,13 @@ export interface NotificationOptions {
      * receiving interaction events from notifications.
      */
     onSelect?: ActionDeclaration<never, never>|null;
+
+    /**
+     * An action that will fire when the notification is closed. This can be from user interaction (until future revisions
```

Comment:
Are we consistent with terminology here? I thought actions were "raised"?

The select action above covers a lot of "general" actions stuff that is omitted here. Not saying we need to copy-paste everything, maybe we move some of this to the top of `actions.ts` and link from both of these. But bear in mind that `onClose` will appear before `onSelect` in the API docs - may not be clear at this point what actions are.

Reply:
✅